website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -46,33 +56,8 @@ By default this will be the namespace that the operator is running in. To watch
 mgr, err := ctrl.NewManager(cfg, manager.Options{Namespace: ""})
 ```
 
-It is also possible to use the [MultiNamespacedCacheBuilder][multi-namespaced-cache-builder] to watch a specific set of namespaces:
```

Comment:
Why remove this?

Reply:
Because this info is not relevant to the tutorial and in 90% of the cases then, shows only bring confusion. 
Also, in the tutorial, we already point here the user to check the kb doc https://deploy-preview-4021--operator-sdk.netlify.app/docs/building-operators/golang/tutorial/#manager for deeper details where it is described as well.

Reply:
This is the Tutorial page, not the quickstart guide, though. I feel OK with leaving this link in here.

Reply:
-  Is this text helpful to users understand how the operators work?  Is this instruction used in the sample?  **No, it is not used at all here**
- Are we describing a common scenario case or an advanced one? It is NOT a common required and is a detailed feature/option that will be used ONLY in advanced cases then if we need to the doc that should be in the advanced section instead which indeed I do not think that is required at all since we keep this info by sharing the link for the kubebuilder documentation.

So, IMO it brings confusion since has none relation with the trial sample at all. However, if we have more people asking for keeping it then, I am happy to revert it.

Reply:
I am closing this as sorted out. Please, feel free to ping me if you think that it should be re-opened.